README.md

```diff
@@ -38,9 +38,69 @@ debug('I am debug')
 error('I am error')
 ```
 
+### MDC - Mapped Diagnostic Context
+Inspired by [Logback's implementation](https://logback.qos.ch/manual/mdc.html) of [SLF4J MDC API](https://www.slf4j.org/manual.html#mdc)
+
+MDC allows you to store extra data which will be automagically appended to every log entry as long as you stay in the current context. In Logz.io, the MDC keys will be shown as Fields and can be used as filter parameters.
+
+#### MDC API
+##### MDC.createContext(next)
+Creates MDC context. Has to be called before calling `put`, otherwise `put` will have no effect.
+
+It is possible to create nested contexts. In that case, all MDC data from parent context will be copied to child context, therefore all modifications to MDC in the child context will be done independently from the parent context.
+
+The `next` parameter is context callback. MDC data will be valid during the whole duration of the callback. This makes it ideal to call this in a middleware function (see [request-id.middleware.js](src/request-id.middleware.js) for example).
+```
+const MDC = require('logzio-node-debug').MDC
```

Comment:
I would provide an example with a middleware function. I myself cannot imagine the usage.

Reply:
Added